lib/util/check-restricted.js

```diff
@@ -39,7 +39,12 @@ class Restriction {
             const negate = raw[0] === "!" && raw[1] !== "("
             const pattern = negate ? raw.slice(1) : raw
             const absolute = path.isAbsolute(pattern)
-            const matcher = new Minimatch(pattern, { dot: true })
+
+            const posix = pattern.replace(/\\/g, "/")
```

Comment:
Thinking this through it may be worth checking to see if this is a windows machine first.

Then doing this change as a path with a space in it is also escaped:
`/Some\ Thing`

Reply:
This looks to be incorrect, it's just a bash escape 🤦‍♂️